internal/controlplane/tasks_test.go

```
@@ -386,57 +386,59 @@ func TestRoutineScheduleUsesFrozenOccurrencePromptAndSkipsMissedInstants(t *test
+	if !serviceErrorCode(err, "task_schedule_prompt_too_large") {
 		t.Fatalf("schedule prompt capacity error = %v", err)
 	}
 }
 
-func TestRoutinesMigrationLeavesOnlyFinalProductTables(t *testing.T) {
+func TestTasksMigrationLeavesOnlyFinalProductTables(t *testing.T) {
 	store := newTestStore(t)
-	for _, table := range []string{"routines", "routine_repositories", "work", "work_targets"} {
+	for _, table := range []string{"tasks", "task_repositories", "runs", "sessions"} {
 		var exists int
 		if err := store.db.QueryRow(`SELECT COUNT(*) FROM sqlite_master WHERE type = 'table' AND name = ?`, table).Scan(&exists); err != nil || exists != 1 {
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).QueryRow must not be called. use (*database/sql.DB).QueryRowContext

(noctx)